src/doc/guide.md

```diff
@@ -6,7 +6,7 @@ about how to use Cargo to develop Rust projects.
 # Why Cargo exists
 
 Cargo is a tool that allows Rust projects to declare their various
-dependencies, and ensure that you’ll always get a repeatable build.
+dependencies and ensure that you’ll always get a repeatable build.
 
 To accomplish this goal, Cargo does four things:
 
```

Comment:
It's slightly wired that the second section of guide is `Converting to cargo`. Almost all projects use cargo nowdays, don't they?

Presumably this section was written when Cargo had just appeared, and maybe it's better to just remove it?